cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server.go

```diff
@@ -328,7 +328,19 @@ func (s *Server) GetAvailablePackageDetail(ctx context.Context, request *corev1.
 		return nil, status.Errorf(codes.Internal, "Unable to retrieve chart: %v", err)
 	}
 
-	availablePackageDetail, err := AvailablePackageDetailFromChart(&chart)
+	if len(chart.ChartVersions) == 0 {
```

Comment:
Is it really an error (I mean, `codes.Internal`), or perhaps it is just a temporary 404-alike error because the assets have not been fetched yet?

Reply:
Yeah, I was wondering when adding this. The reason I chose an internal error is because the internal state is incorrect if this condition is true (since we have the chart in the DB, but without any chart versions - it should never be reached, since the `info` json blob is added atomically).